Tasks/app-store-release/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -37,6 +37,8 @@
   "loc.input.help.screenshotsPath": "Path to the screenshots to publish.",
   "loc.input.label.shouldSubmitForReview": "Submit for Review",
   "loc.input.help.shouldSubmitForReview": "Select to automatically submit the new app version for review after the upload is completed.",
+  "loc.input.label.appSpecificId": "App specific apple Id",
+  "loc.input.help.appSpecificId": "App specific apple Id: this value should be taken from Apple ID property in the App Information section in App Store Connect",
```

Comment:
Could you please prepare changes for documentation updates also?

Reply:
Updated in https://github.com/microsoft/app-store-vsts-extension/pull/172/commits/5dc625ddbf6ae19266fd790956be22888a9c4125